src/content/docs/kubernetes-pixie/kubernetes-integration/installation/reduce-ingest.mdx

```diff
@@ -1,114 +1,257 @@
+
+    This log record shows all the attributes that the [New Relic Kubernetes plugin](/docs/logs/forward-logs/kubernetes-plugin-log-forwarding/) usually captures in its logs:
+
+    ```json
+    [
+    {
+        "cluster_name": "api-test",
+        "kubernetes": {
+        "annotations": {
+            "kubernetes.io/psp": "eks.privileged"
+        },
+        "container_hash": "fryckbos/test@sha256:5b098eaf3c7d5b3585eb10cebee63665b6208bea31ef31a3f0856c5ffdda644b",
+        "container_image": "fryckbos/test:latest",
+        "container_name": "newrelic-logging",
+        "docker_id": "134e1daf63761baa15e035b08b7aea04518a0f0e50af4215131a50c6a379a072",
+        "host": "ip-192-168-17-123.ec2.internal",
+        "labels": {
+            "app": "newrelic-logging",
+            "app.kubernetes.io/name": "newrelic-logging",
+            "controller-revision-hash": "84db95db86",
+            "pod-template-generation": "1",
+            "release": "nri-bundle"
+        },
+        "namespace_name": "nrlogs",
+        "pod_id": "54556e3e-719c-46b5-af69-020b75d69bf1",
+        "pod_name": "nri-bundle-newrelic-logging-jxnbj"
+        },
+        "message": "[2021/09/14 12:30:49] [ info] [engine] started (pid=1)\n",
+        "plugin": {
+        "source": "kubernetes",
+        "type": "fluent-bit",
+        "version": "1.8.1"
+        },
+        "stream": "stderr",
+        "time": "2021-09-14T12:30:49.138824971Z",
+        "timestamp": 1631622649138
+    }
+    ]
+    ```
+
+    This is how the previous log record would look like after enabling `lowDataMode`:
+
+    ```json
+    [
+    {
+        "cluster_name": "api-test",
+        "container_name": "newrelic-logging",
+        "namespace_name": "nrlogs",
+        "pod_name": "nri-bundle-newrelic-logging-jxnbj",
+        "message": "[2021/09/14 12:30:49] [ info] [engine] started (pid=1)\n",
+        "stream": "stderr",
+        "timestamp": 1631622649138
+    }
+    ]
+    ```
+    </Collapser>
+    <Collapser
+        id="nr-pixie-integration"
+        title="New Relic Pixie integration"
+    >
+
+    If you enable `lowDataMode`, the `newrelic-pixie` integration performs heavier sampling on Pixie spans and collect data less often, from 10 seconds to 15 seconds. These are the `lowDataMode` settings:
+
+    ```
+    HTTP_SPAN_LIMIT: 750
+    DB_SPAN_LIMIT: 250
+    COLLECT_INTERVAL_SEC: 15
+    ```
+
+    You can find the default settings for these parameters and others in the [newrelic-pixie-integration](https://github.com/newrelic/newrelic-pixie-integration) Github repo.
+
+    </Collapser>
+</CollapserGroup>
+
+
+## Reduce data ingested by changing the scrape interval [#scrape-interval]
+
+The New Relic Kubernetes integration allows changing the interval at which metrics are gathered from the cluster. This lets you choose the right balance between data resolution and usage. We suggest choosing an interval between 15 and 30 seconds for the best experience.
```

Comment:
We should comment that lowDataMode already sets the scrape interval to 30s

Reply:
Added a tip note.